omod/src/main/webapp/resources/htmlforms/hts/htsInitialTest.html

```diff
@@ -840,30 +860,7 @@
                                     jq("#key-population").hide();
                                 }
 
-                                //show syphilis results if they exist
```

Comment:
why are these removed? does the form work OK both in enter and edit modes with the deletes?